gcn_classic_to_json/notices/GECAM_FLT/__init__.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+from ..GECAM_GND import parse as parse_gecam_gnd
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[27:39]  # Unused. According to docs: '32 bytes for the future'
+    bin[21]  # Temporarily Unused. This is trigger_duration; currently no keyword
+    bin[22:24]  # Unused. Equivalent to rate_energy_range
+
+    detector_options = ["on", "triggered"]
+    detector_bits = np.flip(np.unpackbits(bin[26:27].view(dtype="u1")))
+    detector_status = [detector_options[bit] for bit in detector_bits[:25]]
+    detectors = dict(zip(list(np.arange(25.0) + 1), detector_status))
```

Comment:
```suggestion
    detectors = dict(zip(np.arange(1, 26), detector_status))
```

Reply:
This format gives the following error:
TypeError: keys must be str, int, float, bool or None, not int64

Reply:
Then how about:

```suggestion
    detectors = dict(zip(range(1, 26), detector_status))
```

Reply:
Yup works, I've pushed the new code